### PR TITLE
12.0 dafa afc 2118 interpreter translations

### PR DESCRIPTION
**Pull request by nils-vertel**

### CODE REVIEW

File: task_interpreter_ipf_client/i18n/sv_SE.po [hunk 1]
```diff
@@ -0,0 +1,309 @@
...
+# This file contains the translation of the following modules:
+#	* task_interpreter_ipf_client
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0-20210131\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-04-19 13:35+0000\n"
+"PO-Revision-Date: 2021-04-19 13:35+0000\n"
+"Last-Translator: <>\n"
```
Comment (SebastianGieser-Vertel) [inline task_interpreter_ipf_client/i18n/sv_SE.po]: add name?

File: task_interpreter_ipf_client/i18n/sv_SE.po [hunk 2]
```diff
@@ -0,0 +1,309 @@
...
+#. module: task_interpreter_ipf_client
+#: model:ir.model.fields,field_description:task_interpreter_ipf_client.field_ipf_interpreter_client_test__request_history_ids
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.ipf_client_config_view_form
+msgid "Requests"
+msgstr "Anrop"
+
+#. module: task_interpreter_ipf_client
+#: model:ir.model.fields,field_description:task_interpreter_ipf_client.field_ipf_interpreter_request_history__response_code
+msgid "Response Code"
+msgstr "Svart Kod"
```
Comment (SebastianGieser-Vertel) [inline task_interpreter_ipf_client/i18n/sv_SE.po]: Looks like a typo here

File: task_interpreter_ipf_client/i18n/sv_SE.po [hunk 3]
```diff
@@ -0,0 +1,309 @@
...
+
+#. module: task_interpreter_ipf_client
+#: selection:res.config.settings,ipf_environment:0
+msgid "T2"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: model:ir.model,name:task_interpreter_ipf_client.model_ipf_interpreter_client
+msgid "Task Interpreter IPF client"
+msgstr ""
```
Comment (SebastianGieser-Vertel) [inline task_interpreter_ipf_client/i18n/sv_SE.po]: No translation?

File: task_interpreter_ipf_client/i18n/sv_SE.po [hunk 4]
```diff
@@ -0,0 +1,309 @@
...
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.ipf_client_config_view_form
+msgid "distanstolktyp"
+msgstr ""
+
+#. module: task_interpreter_ipf_client
+#: model_terms:ir.ui.view,arch_db:task_interpreter_ipf_client.ipf_client_config_view_form
+msgid "kon"
```
Comment (SebastianGieser-Vertel) [inline task_interpreter_ipf_client/i18n/sv_SE.po:288]: Should this and the terms below not be in english and then translated?

**Review by SebastianGieser-Vertel — CHANGES_REQUESTED** [review]

File: task_interpreter_ipf_client/i18n/sv_SE.po (hunk 4 above)
Comment (nils-vertel) [reply task_interpreter_ipf_client/i18n/sv_SE.po:288]: Non trivial to fix now, will be fixed in the future

**Pull request merged**